docs/api.yaml

```diff
@@ -60,13 +71,22 @@ paths:
           schema:
             type: string
             example: "W012A3CDE"
+        - in: query
+          name: include
+          schema:
+            nullable: true
+            type: string
+          description: Used to include the optional field `quotes`
+          example: "include=quotes"
       responses:
         "200":
           description: "Person matching criteria:"
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/Person"
+                oneOf:
```

Comment:
The `oneOf` key works somewhat well for list endpoints, but it doesn't seem to work so well for single items.

There's an FAQ at the bottom of this page in the docs asking about mapping request params to responses, and it doesn't seem to be possible at the moment 😢 https://swagger.io/docs/specification/describing-responses/

Reply:
That seems ok? If it's a single item, wouldn't we just use what we were doing before?

Reply:
Yeah, we could just have it show the Person details without the quotes. We just then don't have an example for when choosing to pass the param `include=quotes`